### PR TITLE
DB Complete

### PR DESCRIPTION
**Pull request by astrohlicpu**
Hi,
I completed your business scenario. Please review my work and let me know if you need any changes.

### CODE REVIEW

File: HighSchoolDB/3 Reports.sql [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+-- 2. Show a list of students who have already sent in the full tuition ($15,000)
+SELECT FirstName, LastName, Grade, Class, TuitionReceived
+FROM Students
+WHERE TuitionReceived = 15000;
+
+-- 3. Show a list of girls who have at least one sister in school with them
+select s.LastName as SistersInSchool
+from Students s
+group by s.LastName
```
Comment (wolmanr) [inline HighSchoolDB/3 Reports.sql]: Some students have the same last name but different addresses. They may be cousins but  they are not sisters

File: HighSchoolDB/1 Table.sql [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        LastName  varchar(50) not null constraint ck_Students_LastName_cannot_be_blank check(LastName > ''),
+        Address varchar(100) not null constraint ck_Students_Address_cannot_be_blank check(Address > ''),
+        City varchar(50) not null constraint ck_Students_City_cannot_be_blank check(City > ''),
+        State varchar(50) not null constraint ck_Students_State_cannot_be_blank check(State > ''),
+        ZipCode char(5) not null constraint ck_Students_Zipcode_must_be_5_numbers check(ZipCode like replicate('[0-9]',5)),
+        Grade tinyint not null constraint ck_Students_Grade_must_be_between_9_and_12 check(grade between 9 and 12),
+        Class char(1) not null constraint ck_Students_Class_must_be_AB_or_C check(Class in ('A','B','C')),
+        FavoriteSubject varchar(50) not null constraint ck_Students_FavoriteSubject_cannot_be_blank check(FavoriteSubject > ''),
+        TuitionReceived int not null constraint ck_Students_TuitionReceived_cannot_be_negative_or_above_15000 check(TuitionReceived between 0 and 15000),
+        constraint u_Students_FirstName_LastName_must_be_unique unique(FirstName,LastName)
```
Comment (wolmanr) [inline HighSchoolDB/1 Table.sql:21]: We need another `column` stating the student's remaining balance in her tuition, as stated in the spec.

**Review by wolmanr — CHANGES_REQUESTED** [review]
I reviewed the work. You did I great job. I noticed 2 things missing, please see my inline comments

File: HighSchoolDB/3 Reports.sql (hunk 1 above)
Comment (astrohlicpu) [reply HighSchoolDB/3 Reports.sql]: Updated

File: HighSchoolDB/1 Table.sql (hunk 2 above)
Comment (astrohlicpu) [reply HighSchoolDB/1 Table.sql:21]: Updated

File: HighSchoolDB/3 Reports.sql (hunk 1 above)
Comment (wolmanr) [reply HighSchoolDB/3 Reports.sql]: look great! thanks

File: HighSchoolDB/1 Table.sql (hunk 2 above)
Comment (wolmanr) [reply HighSchoolDB/1 Table.sql:21]: looks great! thanks

**Pull request closed without merging**